pytest.ini

```diff
@@ -19,24 +19,13 @@
 #    updated.
 # 3) Ignore DeprecationWarning for datetime.utcfromtimestamp() triggered
 #    from dateutil. See https://github.com/dateutil/dateutil/issues/1314.
-# 4 & 5) The pyOpenSSL X509/PKey warnings are due to probe_sni and/or tests for
```

Comment:
I am pretty sure this was incorrect, and it was actually just used by SSLSocket. Though `probe_sni` tests did use SSLSocket so that could be what this meant.